docs/aa_integration.md

```diff
@@ -1,34 +1,34 @@
-# AnsibleApp Integration
+# Ansible Playbook Bundles (APB) Integration
 
-## Running AnsibleApps with the ASB
+## Running APB with the ASB
```

Comment:
Keeping track of these two acronyms when referenced closely is going to get difficult :/